<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>981</width>
      <height>641</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>page1_bg.webp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>891</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: rgba(255, 255, 255, 0.975); /* White background with 80% opacity */
    border-radius: 50px; /* Rounded corners */
    padding: 20px; /* Optional: Add some padding inside the box */
    margin: 20px; /* Optional: Add margin around the box */
    color: #000000; /* Text color inside the box */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">FONT-SIZE:30PX;</string>
    </property>
    <property name="text">
     <string>VELOCITY CALCULATOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>490</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #5bb7c6, stop:0.7 #e3655b);
    border-radius: 25px;
    border: none;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    box-shadow: 0px 5px 15px rgba(0, 0, 0, 0.3);
    transition: all 0.3s ease;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #e3655b, stop:1 #5bb7c6);
    transform: scale(1.05);
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #c94d3f, stop:1 #497d87);
    transform: scale(0.95);
    box-shadow: 0px 3px 10px rgba(0, 0, 0, 0.2);
}
</string>
    </property>
    <property name="text">
     <string>SOLVE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>v=</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #F0F0F0;
    color: #2E2E2E;
    border: 2px solid #B0B0B0;
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
    padding: 5px;
    font-size: 18px;
    text-align: center;
}

QLineEdit:hover {
    border: 2px solid #8A8A8A;
    background-color: #F5F5F5;
}

QLineEdit:focus {
    border: 2px solid #6A6A6A;
    background-color: #E0E0E0;
}

QLineEdit::placeholder {
    color: #A0A0A0;
    font-size: 14px;
    text-align: center;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>d</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #F0F0F0;
    color: #2E2E2E;
    border: 2px solid #B0B0B0;
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
    padding: 5px;
    font-size: 18px;
    text-align: center;
}

QLineEdit:hover {
    border: 2px solid #8A8A8A;
    background-color: #F5F5F5;
}

QLineEdit:focus {
    border: 2px solid #6A6A6A;
    background-color: #E0E0E0;
}
QLineEdit::placeholder {
    color: #A0A0A0;
    font-size: 14px;
    text-align: center;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>t</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>———</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>220</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>v =  velocity</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>280</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>d =</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>280</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FAFAFA;
    color: #333333;
    border: 2px solid #CCCCCC;
    border-top-left-radius: 12px;
    border-top-right-radius: 12px;
    border-bottom-left-radius: 12px;
    border-bottom-right-radius: 12px;
    padding: 6px;
    font-size: 16px;
    text-align: center;
}

QLineEdit:hover {
    border: 2px solid #AAAAAA;
    background-color: #FFFFFF;
}

QLineEdit:focus {
    border: 2px solid #888888;
    background-color: #F2F2F2;
}

QLineEdit::placeholder {
    color: #B0B0B0;
    font-size: 15px;
    text-align: center;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>displacement</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>340</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>t =</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FAFAFA;
    color: #333333;
    border: 2px solid #CCCCCC;
    border-top-left-radius: 12px;
    border-top-right-radius: 12px;
    border-bottom-left-radius: 12px;
    border-bottom-right-radius: 12px;
    padding: 6px;
    font-size: 16px;
    text-align: center;
}

QLineEdit:hover {
    border: 2px solid #AAAAAA;
    background-color: #FFFFFF;
}

QLineEdit:focus {
    border: 2px solid #888888;
    background-color: #F2F2F2;
}

QLineEdit::placeholder {
    color: #B0B0B0;
    font-size: 15px;
    text-align: center;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>page1_formula.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
